test/format/util.c

```diff
@@ -78,19 +78,26 @@ key_gen_setup(WT_ITEM *key)
 }
 
 static void
-key_gen_common(WT_ITEM *key, uint64_t keyno, int suffix)
+key_gen_common(WT_ITEM *key, uint64_t keyno, const char * const suffix)
 {
 	int len;
 	char *p;
 
 	p = key->mem;
 
 	/*
-	 * The key always starts with a 10-digit string (the specified cnt)
-	 * followed by two digits, a random number between 1 and 15 if it's
-	 * an insert, otherwise 00.
+	 * The key always starts with a 10-digit string (the specified row
+	 * number), followed by a suffix if it's an insert.
 	 */
-	len = sprintf(p, "%010" PRIu64 ".%02d", keyno, suffix);
+	u64_to_string_zf(keyno, key->mem, 11);
+	if (suffix == NULL)
+		len = 10;
```

Comment:
Does it matter that this is functionally different to the previous implementation? The old version always created the same length strings, this one will create strings with a different length depending on whether the suffix is NULL.